lms/envs/common.py

```diff
@@ -801,9 +801,6 @@
     # .. toggle_tickets: https://github.com/edx/edx-platform/pull/15006
     'ENABLE_BULK_ENROLLMENT_VIEW': False,
 
-    # Whether course goals is enabled.
-    'ENABLE_COURSE_GOALS': True,
```

Comment:
There's no situation where this would affect anyone in open edx land, right? This is always checked in concert with the waffle?

Reply:
yup